### ISSUE TITLE
No files/directories in */pip-egg-info (from PKG-INFO)

### ISSUE THREAD

**Issue by Nelyah** [mitsuhiko/pipsi#134]
Not sure where the error comes from. I am able to install other packages with pipsi and I am also able to install this package in particular (`prody`) with `pip install --user prody`

    $ pipsi install prody
    Already using interpreter /home/chloe/.local/venvs/pipsi/bin/python
    Using base prefix '/usr'
    New python executable in /home/chloe/.local/venvs/prody/bin/python
    Installing setuptools, pip, wheel...done.
    Collecting prody
      Using cached ProDy-1.9.4.tar.gz
    No files/directories in /tmp/pip-build-i61j2ml8/prody/pip-egg-info (from PKG-INFO)
    Failed to pip install.  Aborting.

I search through [their github](https://github.com/prody/ProDy) repo and found that they have the following lines in their .gitignore:

     *.egg
     ´.egg-info

I also tried to install prody with the following command `pipsi install git+https://github.com/prody/ProDy/#egg=prody` but I get the same error as above. 

Any insight would be appreciated!

**Comment by Nelyah** [comment]
Hi,
Sadly no, I simply ended up using the `pip install --user prody` version

**Comment by bunmalik** [comment]
Thanks @Nelyah 
It still hasnt work for me.

**Comment by rkantos** [comment]
I was able to install seasnake (per user) after installing seasnake manually and also following their llvm env install instructions: https://github.com/pybee/sealang

**Comment by marimeireles** [comment]
Yeah, I ran in to the same problem in my Fedora 29, but not on my MacOS. So maybe is just something related to Linux? It's worth it to give it a try if you can run it in a different OS.
